### ISSUE TITLE
[Homebrew] Warning: No checksum defined for cask 'xpipe', skipping verification

### ISSUE THREAD

**Issue by TonyBostonTB** [xpipe-io/xpipe#252]
When trying to install xpipe, I get the warning from homebrew, that no checksum was defined for xpipe. Is this wanted behaviour?

```
Warning: No checksum defined for cask 'xpipe', skipping verification.
==> Installing dependencies: util-linux
==> Downloading https://ghcr.io/v2/homebrew/core/util-linux/manifests/2.39.3-1
################################################################################################################################################################################## 100.0%
==> Fetching util-linux
==> Downloading https://ghcr.io/v2/homebrew/core/util-linux/blobs/sha256:c0fe29bd7ce098ba80b119b4a6b37c16255e9a028724f91ca64efb5259fc9a34
################################################################################################################################################################################## 100.0%
==> Installing util-linux
==> Pouring util-linux--2.39.3.arm64_sonoma.bottle.1.tar.gz
🍺  /opt/homebrew/Cellar/util-linux/2.39.3: 229 files, 16.3MB
==> Installing Cask xpipe
==> Running installer for xpipe with sudo; the password may be necessary.
Password:
```
I'm just unsure if I want to give it the password when this checksum error comes up before. I know why the password is needed, anyway.

**Comment by crschnick** [comment]
Yeah that is a good point, I will fix that.

The original reason why the checksum validation is skipped is that the ARM builds were built separately and not in the release pipeline due to Github actions not having ARM runniners, so the checksums were not available when the homebrew package was published. But they are now ARM runners available, so I should be able to compute to checksum.

If you feel uneasy with that, you can download the manual releases like a .dmg or .pkg, they have a checksum attached in the release artifacts.

**Comment by TonyBostonTB** [comment]
Thanks @crschnick - understood. Should we keep the issue open then?

**Comment by crschnick** [comment]
yes

**Comment by tacaly** [comment]
Fixed in release version 9.2: https://github.com/xpipe-io/xpipe/releases/tag/9.2
